packages/application-to-register-webapp/src/utils/helpers.js

```diff
@@ -57,11 +59,20 @@ const getReferrer = (request, referrerId, unsetFlag) => {
   return currentReferrer
 }
 
+const getRegistrationTasks = request => {
```

Comment:
Why does the whole request object need to be passed to the function, surely only the value for registrationTasks?

Reply:
We are passing the whole request so that we can get yar object in one location and not duplicate the code

Reply:
It needs the request. Please look at the code.

Reply:
I have looked at the code as a whole and I disagree, functional it makes no difference so be it.